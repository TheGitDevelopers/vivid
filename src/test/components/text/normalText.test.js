import React from 'react';
import renderer from 'react-test-renderer';
import NormalText from '../../../components/common/text/normalText';


describe('NormalText tests:', () => {
  /* Test mockups */
  const tree = renderer
    .create(<NormalText />);

  /* All component tests should be listed here */
  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });

  it('has "text" prop which default is null', () => {
    expect(tree.root.find(NormalText).props.text).toBe(null);
  });
});
