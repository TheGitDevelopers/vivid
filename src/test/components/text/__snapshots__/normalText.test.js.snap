// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NormalText tests: renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#A6BCD0",
        "fontFamily": "lato-regular",
        "fontSize": 16,
      },
      null,
    ]
  }
/>
`;
